android/app/src/main/res/layout/fragment_parent_clan.xml

```diff
@@ -24,18 +24,25 @@
     <android.support.v7.widget.RecyclerView
         android:id="@+id/child_recycler_view"
         android:layout_width="match_parent"
-        android:layout_height="450dp"
+        android:layout_height="match_parent"
         android:layout_below="@+id/parent_recycler_view"
         android:layout_marginTop="10dp"
         android:scrollbars="vertical" />
 
-    <!-- TODO: convert to image button -->
-    <Button
-        android:id="@+id/add_clan_member_button"
+
+    <android.support.design.widget.CoordinatorLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_below="@id/child_recycler_view"
-        android:text="@string/add_a_clan_member"
-        android:textSize="30sp" />
+        android:layout_height="match_parent">
+
+        <android.support.design.widget.FloatingActionButton
+            android:id="@+id/add_clan_member_fab"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_centerHorizontal="true"
+            android:layout_gravity="bottom|end"
+            android:layout_margin="15dp"
+            android:src="@mipmap/ic_launcher" /> <!-- TODO: change icon -->
```

Comment:
wanna just change this to @drawable/plus_bw?

Reply:
so if we set this to `@drawable/plus_bw`
it will be confusing for the user because this same image is currently being used for parent's `add` tab as well

also I think we should change `Add` to `Create`

Reply:
good point we can use something like this?
![image](https://user-images.githubusercontent.com/25283324/60224286-c7985b80-984f-11e9-8505-844e6995bc3e.png)


Agreed `Create` is better

Reply:
I +1 having `Create` instead of `Add`, sounds more Jim-like.
Why don't we just have the default Android plus button for now, that sword seems very out of place.
As long as we change this later I'm okay with merging this now.